app/allocation/views/view.njk

```diff
@@ -31,111 +40,100 @@
+      }) }}
     </h1>
   </header>
 {% endblock %}
 
-{% macro emptySlot(content) %}
-  <div class="app-border-top-1">
-    {{ appMessage({
-      classes: "app-message--muted govuk-!-margin-top-3 govuk-!-margin-bottom-3 govuk-!-padding-right-8",
-      allowDismiss: false,
-      content: {
-        html: content
-      }
-    }) }}
-  </div>
-{% endmacro %}
-
 {% block contentMain %}
   <section>
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.criteria_for_allocation") }}</h2>
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
+      {{ t("allocation::view.criteria.heading") }}
+    </h2>
+
     <span class="govuk-caption-m govuk-!-margin-bottom-4">
-      {{ t("allocations::view.created_at") }}
+      {{ t("created_on") }}
       {{ allocation.created_at | formatDateAsRelativeDay }}
       {{ allocation.created_at | formatTime }}
     </span>
 
-    {{ govukSummaryList(allocationDetails) }}
+    {{ govukSummaryList(criteria) }}
 
-    {% if allocation.complete_in_full and allocationPeople.emptySlots %}
+    {% if allocation.complete_in_full %}
       {{ govukWarningText({
-        text: t("allocations::view.complete_in_full"),
+        text: t("allocation::view.criteria.complete_in_full"),
         iconFallbackText: "Warning"
       }) }}
     {% endif %}
   </section>
 
+  <section class="govuk-!-margin-top-8">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
+      {{ t("allocation::view.people.heading") }}
+    </h2>
 
-
-  {% if canAccess('allocation:person:assign') %}
-    <section class="govuk-!-margin-top-8">
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.people_assign.heading") }}</h2>
-    {% if allocationPeople.emptySlots %}
-    <p>{{ t("allocations::view.people_assign.special_vehicle_instruction", {href:"/move/new"}) | safe }}</p>
-
-    <h3 class="govuk-heading-s">{{ t("allocations::view.people_assign.slots_empty", {
-      slots_empty: allocationPeople.emptySlots,
-      person_or_people: t('person', {
-        count: allocationPeople.emptySlots
-      })
-    }) }}</h3>
-      <p>{{ govukButton({
-        html: "Add person",
-        href: "/move/" + unassignedMoveId + "/assign"
-      }) }}</p>
-    {% endif %}
-    {% if allocationPeople.filledSlots.length === 0 %}
-      {{ emptySlot(t("allocations::view.people_assign.no_slots_filled")) }}
+    <h3 class="govuk-heading-s">
+      {{ t("allocation::view.people.progress", {
+        count: remainingCount if canAccess('allocation:person:assign') else totalCount,
+        addedCount: addedCount,
+        context: "remaining" if canAccess('allocation:person:assign')
+      }) }}
+    </h3>
+
+    {% if canAccess('allocation:person:assign') and remainingCount %}
+      <p>
+        {{ t("allocation::view.people.special_vehicle_instruction", {
+          href:"/move/new"
+        }) | safe }}
+      </p>
+
+      {{ govukButton({
+        href: "/move/" + unassignedMoveId + "/assign",
```

Comment:
Really unimportant, but maybe use template strings.

Reply:
Unfortunately it's not possible as this is a nunjucks file